spec/intl.html

```diff
@@ -1340,6 +1340,270 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
             1. Return _result_.
           </emu-alg>
         </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatetoiso" type="abstract operation">
+          <h1>
+            CalendarDateToISO (
+              _calendar_: a String,
+              _fields_: a Record,
+              _overflow_: an Object,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to convert _fields_ that describes a date in the calendar represented by the string _calendar_ to the ISO calendar and returns the corresponding day, month and year values as a Record with the fields [[Day]], [[Month]] and [[Year]] respectively.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateaddition" type="abstract operation">
+          <h1>
+            CalendarDateAddition (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+              _duration_: a %Temporal.Duration%,
+              _overflow_: an Object,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to add _duration_ to _date_ in the context of the calendar represented by the string _calendar_ and returns the corresponding day, month and year of the result in the ISO calendar values as a Record with the fields [[Day]], [[Month]] and [[Year]] respectively.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedifference" type="abstract operation">
+          <h1>
+            CalendarDateDifference (
+              _calendar_: a String,
+              _one_: a %Temporal.PlainDate%,
+              _two_: a %Temporal.PlainDate%,
+              _largestUnit_: a String,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to find the difference between the two dates _one_ and _two_ in the context of the calendar represented by the string _calendar_ and returns the corresponding years, months, weeks and days as a Record with the fields [[Years]], [[Months]], [[Weeks]] and [[Days]] respectively.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateera" type="abstract operation">
+          <h1>
+            CalendarDateEra (
+              _calendar_: a String,
+              _datetime_: an Object with the [[ISOYear]] internal slot,
```

Comment:
Here and below: a calendar is going to need all three of [[ISOYear]], [[ISOMonth]], [[ISODay]] internal slots to be able to determine any field values in calendar space.

Also I'd recommend naming the parameter _date_ or _dateLike_ instead of _datetime_.

Reply:
Done and done